resources/schemas/dbscripts/postgresql/cds-22.005-22.006.sql

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2022 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+--  Update cds.import_studyReport table
+ALTER TABLE cds.import_studyReport ADD COLUMN cds_report_link TEXT;
+ALTER TABLE cds.import_studyReport ADD COLUMN cds_report_label TEXT;
```

Comment:
seems fine but do we anticipate needing to store information in excess of say 250 characters?

Reply:
done